.travis.yml

```diff
@@ -22,7 +23,7 @@ deploy:
     on:
       tags: true
   - provider: script
-    script: sh bin/publish-to-dockerhub.sh
```

Comment:
Means the script is run with what's specified in the shebang (bash) and not `sh`.